sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -317,11 +317,20 @@ export class CallbackFunction<E, R> extends LambdaFunction {
             ...args,
             code: new pulumi.asset.AssetArchive(codePaths),
             handler: serializedFileNameNoExtension + "." + handlerName,
-            runtime: args.runtime || runtime.NodeJS8d10Runtime,
+            runtime: args.runtime || runtime.NodeJS10dXRuntime,
```

Comment:
BTW - should we consider jumping to `nodejs12.x` directly - since Node 12 is already the LTS?

Reply:
Done! Great point - that was my missing that it was the LTS!